bootstrap/api/v1beta1/rke2config_types.go

```diff
@@ -156,6 +156,11 @@ type RKE2AgentConfig struct {
 	// basically supposing that online container registries and RKE2 install scripts are not reachable.
 	AirGapped bool `json:"airGapped,omitempty"`
 
+	// AirGappedChecksum is a string value with a sha256sum checksum to compare with checksum
+	// of existing sha256sum-<arch>.txt file for packages already available on the machine
+	// before performing air-gapped installation.
+	AirGappedChecksum string `json:"airGappedChecksum,omitempty"`
```

Comment:
can we make this field optional?

Reply:
It was blocked on #313 for fuzzy testing, so I was only able to add generated manifests, make the field optional and backward value recovery after it got merged. This should be ready for review.